core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -111,8 +111,28 @@ void SYCL::print_configuration(std::ostream& os, bool verbose) const {
   os << "macro  KOKKOS_IMPL_SYCL_USE_IN_ORDER_QUEUES : undefined\n";
 #endif
 
-  if (verbose)
+#ifdef KOKKOS_ENABLE_ONEDPL
+  os << "macro  KOKKOS_ENABLE_ONEDPL : defined\n";
+#else
+  os << "macro  KOKKOS_ENABLE_ONEDPL : undefined\n";
+#endif
+
+  int counter       = 0;
+  int active_device = Kokkos::device_id();
+  std::cout << "\nAvailable devices: \n";
+  std::vector<sycl::device> devices = Impl::get_sycl_devices();
+  for (const auto& device : devices) {
+    os << "[" << device.get_backend() << "]:gpu:" << counter << "] "
```

Comment:
Are you sure you always want to print "gpu"?
What about `device->is_gpu() : "gpu" ? ...`

Reply:
`get_sycl_devices` only ever returns GPU devices so the output is consistent at least.